### PR TITLE
Add namespace to requests where it was missing

### PR DESCRIPTION
**Pull request by vitarb**
Similarly to [this PR](https://github.com/temporalio/sdk-go/pull/304) in go SDK, we are populating namespace field where it was missing.

Related proto changes in the API repo are [#101](https://github.com/temporalio/api/pull/101) and [#102](https://github.com/temporalio/api/pull/102).

### CODE REVIEW

**Review by sergeybykov — CHANGES_REQUESTED** [review]
Need to also set namespace at https://github.com/temporalio/sdk-java/blob/30420212a26283cf7cbe6339512058a3186074e8/temporal-sdk/src/main/java/io/temporal/internal/worker/PollWorkflowTaskDispatcher.java#L82 and https://github.com/temporalio/sdk-java/blob/30420212a26283cf7cbe6339512058a3186074e8/temporal-sdk/src/main/java/io/temporal/internal/external/ManualActivityCompletionClientImpl.java#L104?

**Comment by sergeybykov** [comment]
Need to also set namespace at https://github.com/temporalio/sdk-java/blob/30420212a26283cf7cbe6339512058a3186074e8/temporal-sdk/src/main/java/io/temporal/internal/worker/PollWorkflowTaskDispatcher.java#L82 and https://github.com/temporalio/sdk-java/blob/30420212a26283cf7cbe6339512058a3186074e8/temporal-sdk/src/main/java/io/temporal/internal/external/ManualActivityCompletionClientImpl.java#L104?

File: temporal-sdk/src/main/java/io/temporal/internal/external/ManualActivityCompletionClientImpl.java [hunk 1]
```diff
@@ -217,6 +208,9 @@ public void recordHeartbeat(Object details) throws CanceledFailure {
       RecordActivityTaskHeartbeatRequest.Builder request =
           RecordActivityTaskHeartbeatRequest.newBuilder()
               .setTaskToken(ByteString.copyFrom(taskToken));
+      if (namespace != null) {
```
[the remark this answers is not in the document]
Comment (vitarb) [reply temporal-sdk/src/main/java/io/temporal/internal/external/ManualActivityCompletionClientImpl.java:211]: Yes, for async activities.

[the remark this answers is not in the document]
Comment (vitarb) [reply temporal-sdk/src/main/java/io/temporal/internal/external/ManualActivityCompletionClientImpl.java:211]: I was able to propagate it to the second constructor, so it shouldn't be ever null now.

**Review by sergeybykov — APPROVED** [review]

**Pull request merged**